replica/distributed_loader.cc

```diff
@@ -219,8 +219,10 @@ distributed_loader::process_upload_dir(distributed<replica::database>& db, shard
         const auto& erm = db.local().find_keyspace(ks).get_vnode_effective_replication_map();
         auto owned_ranges_ptr = skip_cleanup ? lw_shared_ptr<dht::token_range_vector>(nullptr) : compaction::make_owned_ranges_ptr(db.local().get_keyspace_local_ranges(erm).get());
         reshard(directory, db, ks, cf, make_sstable, owned_ranges_ptr).get();
+        if (!skip_reshape) {
         reshape(directory, db, sstables::reshape_mode::strict, ks, cf, make_sstable,
                 [] (const sstables::shared_sstable&) { return true; }).get();
```

Comment:
You can fix the indentation, normally we recommend this be done in a separate commit, but I think people will be able to deal with 2 additional lines of change.